fab/pillows/swiss.yml

```diff
@@ -5,3 +5,5 @@ exclude_pillows:
   - ReportCaseToElasticsearchPillow
   - ReportXFormToElasticsearchPillow
   - CaseSearchToElasticsearchPillow
+  - kafka-ucr-static
+  - kafka-ucr-main
```

Comment:
How hard would it be to get them to run these services with `pillow-key` instead of `pillow-name`, or even just one `run_ptop` service for all pillows? I doubt having all of these services separate matters for  low volume server